GWAS/LMM.ipynb

```diff
@@ -777,11 +781,11 @@
    "outputs": [],
    "source": [
     "# Select the SNPs and samples to be used based on maf, geno, hwe and mind options\n",
-    "[regenie_qc: provides = [f'{cwd}/cache/{bfile:bn}.qc_pass.id', f'{cwd}/cache/{bfile:bn}.qc_pass.snplist']]\n",
+    "[regenie_0: provides = [f'{cwd}/cache/{bfile:bn}.qc_pass.id', f'{cwd}/cache/{bfile:bn}.qc_pass.snplist']]\n",
```

Comment:
please don't call it `regenie_0`. Doing so tells SoS that `regenie_0` has to be executed before `regenie_1` which is not the case. `regenie_qc` will be triggered in `regenie_1` on need basis is my impression.